content/zh-cn/blog/contributor1.md

```
@@ -0,0 +1,35 @@
+
+[<font style="color:rgb(31, 35, 40);">祁晓波</font>](https://github.com/qixiaobo)<font style="color:rgb(31, 35, 40);"> </font><font style="color:rgb(31, 35, 40);">作为F6汽车科技基础设施负责人，主要研究微服务、可观测、稳定性、研发效能、java中间件等领域</font>
+
+<font style="color:rgb(31, 35, 40);">自加入 Koupleless (原 SOFAServerless) 项目以来，他贡献了诸多框架与中间件在多模块模式下的最佳实践和治理方案，并从框架角度一起探讨如何以更标准化、可持续性的解决思路和方案，也贡献了 dubbo、apollo 等的 adapter 组件。</font>
+
+<font style="color:rgb(31, 35, 40);">2023 年 12 月26日，Koupleless (原 SOFAServerless) 社区PMC之一 赵真灵 代表 Koupleless (原 SOFAServerless) 社区，宣布 祁晓波</font><font style="color:rgb(31, 35, 40);"> </font>[<font style="color:rgb(31, 35, 40);">@qixiaobo</font>](https://github.com/qixiaobo)<font style="color:rgb(31, 35, 40);"> </font><font style="color:rgb(31, 35, 40);">通过投票，成为社区优秀 Contributor！</font>
+
+### <font style="color:rgb(31, 35, 40);">commits 记录/PR 贡献等：</font>
+[<font style="color:rgb(31, 35, 40);">https://github.com/sofastack/sofa-serverless/pulls?q=is%3Apr+is%3Aclosed+author%3Aqixiaobo</font>](https://github.com/sofastack/sofa-serverless/pulls?q=is%3Apr+is%3Aclosed+author%3Aqixiaobo)<font style="color:rgb(31, 35, 40);">  
+</font>
+
+![](https://intranetproxy.alipay.com/skylark/lark/0/2024/png/149473/1727419260641-0f08d99d-2726-4af0-9534-eb659d7c8527.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)